### PR TITLE
Path to vault binary moved.

### PR DESCRIPTION
**Pull request by amouat**
The path to vault in the underlying package will be moved by https://github.com/wolfi-dev/os/pull/2321

This update corrects the path but *does not* add the fixes for upstream helm chart compatibility (we'll do that separately).

Don't merge this until https://github.com/wolfi-dev/os/pull/2321 lands (will mark draft).

### CODE REVIEW

**Review by imjasonh — APPROVED** [review]
lgtm once the package lands

**Comment by amouat** [comment]
Let's not do this. Instead I propose we move straight to the new version in https://github.com/chainguard-images/images/pull/752

The problem is just running `docker run cgr.dev/chainguard/vault` on this version will fail with `Operation not permitted` and no further information if `--cap-add IPC_LOCK` isn't passed.

**Pull request closed without merging**
